app/controllers/api/v3/public/channels_controller.rb

```diff
@@ -11,6 +11,39 @@ def totals
     render(json: statistical_totals_json, status: 200)
   end
 
+  # Takes an array of channel identifiers and retuns a dictionary of channel identifiers as keys and true/false as values
+  def allowed_countries
+    channels = params[:channel_ids].map do |id|
+      {channel: Channel.find_by_channel_identifier(id), channel_identifier: id}
+    end
+
+    parameters = Rewards::Parameters.new.get_parameters
```

Comment:
This is getting repetitive, will look to clean it up in the upcoming related tickets.